website/docs/targets.md

```diff
@@ -11,29 +11,73 @@ Many components, such as package dependencies and base images, are specific to a
 To print a list of available build targets:
 
 ```shell
-GET                           DESCRIPTION
+$ docker buildx build --call targets --build-arg ghcr.io/azure/dalec/frontend:latest - <<< "null"
+TARGET                           DESCRIPTION
 azlinux3/container (default)     Builds a container image for
 azlinux3/container/depsonly      Builds a container image with only the runtime dependencies installed.
 azlinux3/rpm                     Builds an rpm and src.rpm.
 azlinux3/rpm/debug/buildroot     Outputs an rpm buildroot suitable for passing to rpmbuild.
 azlinux3/rpm/debug/sources       Outputs all the sources specified in the spec file in the format given to rpmbuild.
 azlinux3/rpm/debug/spec          Outputs the generated RPM spec file
 azlinux3/worker                  Builds the base worker image responsible for building the rpm
+bionic/deb (default)             Builds a deb package.
```

Comment:
i am confused how can we have different defaults?

Reply:
Its default for the distro.
We could just not set this but perhaps in a separate change.